config/config.py.template

```diff
@@ -675,6 +675,19 @@ if MLLIB_SPARK_VERSION >= 1.3:
     MLLIB_TESTS += [("fp-growth", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
         MLLIB_JAVA_OPTS, [ConstantOption("fp-growth")] + MLLIB_FP_GROWTH_TEST_OPTS)]
 
+MLLIB_PREFIX_SPAN_TEST_OPTS = MLLIB_FPM_TEST_OPTS + \
+                            [OptionSet("num-sequences", [5000000], can_scale=True),
```

Comment:
What is the expected running time? We try to control the running time to be not too short (couple seconds) and not too long (more than 1 minute). If we don't know yet, we can leave a TODO.